### PR TITLE
Allow custom items to be given to all players at once

### PR DESCRIPTION
**Pull request by 0b10000**
The title explains most of this PR. I also added a check to exclude giving cuffed players custom items.

### CODE REVIEW

File: Exiled.CustomItems/Commands/Give.cs [hunk 1]
```diff
@@ -50,43 +50,54 @@ public bool Execute(ArraySegment<string> arguments, ICommandSender sender, out s
 
             if (arguments.Count < 2)
             {
-                response = "give [Custom item name/Custom item ID] [Nickname/PlayerID/UserID]";
+                response = "give [Custom item name/Custom item ID] [Nickname/PlayerID/UserID/*]";
```
Comment (iopietro) [inline Exiled.CustomItems/Commands/Give.cs:53]:
```suggestion
                response = "give [Custom item name/Custom item ID] [Nickname/PlayerID/UserID/all/*]";
```

File: Exiled.CustomItems/Commands/Give.cs [hunk 2]
```diff
@@ -50,43 +50,54 @@ public bool Execute(ArraySegment<string> arguments, ICommandSender sender, out s
...
-            {
-                response = $"Custom item with ID {arguments.At(0)} given to {player.Nickname} ({player.UserId})";
-                return true;
-            }
-            else if (CustomItem.TryGive(player, arguments.At(0)))
+            switch (identifier)
             {
-                response = $"Custom item {arguments.At(0)} given to {player.Nickname} ({player.UserId})";
-                return true;
+                case "*":
```
Comment (iopietro) [inline Exiled.CustomItems/Commands/Give.cs:67]:
```suggestion
                case "*":
                case "all":
```

File: Exiled.CustomItems/Commands/Give.cs [hunk 3]
```diff
@@ -50,43 +50,54 @@ public bool Execute(ArraySegment<string> arguments, ICommandSender sender, out s
...
+                    var eligiblePlayers = Player.List.Where(CheckEligible).ToList();
+                    foreach (var ply in eligiblePlayers)
+                    {
+                        item.Give(ply);
+                    }
```
Comment (iopietro) [inline Exiled.CustomItems/Commands/Give.cs:72]:
```suggestion
                    foreach (var player in eligiblePlayers)
                        item.Give(player);
```

**Review by iopietro — CHANGES_REQUESTED** [review]

**Review by iopietro — APPROVED** [review]

**Review by iRebbok — CHANGES_REQUESTED** [review]
Address those build errors.

**Review by iRebbok — APPROVED** [review]
Approb

**Pull request merged**